appinventor/components/src/com/google/appinventor/components/runtime/ImageSprite.java

```diff
@@ -263,4 +264,14 @@ public double Y() {
   public void MoveTo(double x, double y) {
     super.MoveTo(x, y);
   }
+
+  /**
+   * Moves the %type% so that its left top corner is at the specified x and y coordinates.
+   *
+   * @param coordinates a list of length 2 where the first item is the x-coordinate and the
+   * second item is the y-coordinate.
+   */
+  public void MoveToPoint(YailList coordinates) {
```

Comment:
You don't need to override the super here. That's just a trick we use when we want to expose functions on some components but not others.